app.py

```
@@ -0,0 +1,39 @@
+import streamlit as st
+
+# 각 모듈에서 run 함수 import
+from pages.samples.plotting_sample import run as show_plotting
```

Ruff (0.8.2)

`pages.samples.plotting_sample.run` imported but unused

Remove unused import: `pages.samples.plotting_sample.run`

(F401)